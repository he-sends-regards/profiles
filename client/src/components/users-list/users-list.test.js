import React from 'react';
import renderer from 'react-test-renderer';
import UsersList from './users-list.jsx';

it('Renders correctly when is active', () => {
  const tree = renderer
      .create(
          <UsersList
            isActive={true}
          />,
      )
      .toJSON();
  expect(tree).toMatchSnapshot();
});

it('Renders correctly when is not active', () => {
  const tree = renderer
      .create(
          <UsersList
            isActive={false}
          />,
      )
      .toJSON();
  expect(tree).toMatchSnapshot();
});
