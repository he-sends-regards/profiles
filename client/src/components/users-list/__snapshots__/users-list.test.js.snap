// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
/>
`;

exports[`Renders correctly when is active 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
/>
`;

exports[`Renders correctly when is not active 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
/>
`;
